eo-runtime/src/main/java/org/eolang/PhPackage.java

```diff
@@ -56,26 +56,10 @@ final class PhPackage implements Phi {
 
     @Override
     public Attr attr(final String name) {
-        final StringBuilder abs = new StringBuilder(0).append(this.pkg);
-        if (abs.length() > 0) {
-            abs.append('.');
-        }
-        abs.append(name);
-        final String target = abs.toString()
-            .replaceAll("(^|\\.)([^.]+)", "$1EO$2")
-            .replace("$", "$EO")
-            .replace("-", "_");
         return new AtSimple(
             this.objects.computeIfAbsent(
-                target, t -> {
-                    Phi phi;
-                    try {
-                        phi = this.sub(t);
-                    } catch (final ClassNotFoundException ex) {
-                        phi = new PhPackage(abs.toString());
-                    }
-                    return phi;
-                }
+                this.javaPackage(name),
+                t -> this.loadPhi(t).orElseGet(() -> new PhPackage(this.eoPackage(name)))
```

Comment:
@volodya-lombrozo As far as I understand, here we are using the creation of `pkg` + `name` string two times now. It is not a big deal, but I want to clarify this just for me.

Reply:
@Graur Yes, you right. I though it will be invisible for performance, but if you insist, I can refactor it. What do you think?

Reply:
@volodya-lombrozo No, it's ok. I think we can eave it as is